packages/fast-components-styles-msft/src/divider/index.ts

```diff
@@ -0,0 +1,11 @@
+import {IDesignSystem} from "../design-system";
+import {ComponentStyles} from "@microsoft/fast-jss-manager";
+import {IDividerClassNameContract} from "@microsoft/fast-components-class-name-contracts";
+
+const styles: ComponentStyles<IDividerClassNameContract, IDesignSystem> = {
+    divider: {
+
```

Comment:
we don't add vertical spacing or anything to divider?

Reply:
@chrisdholt  and I had discussed this and think the spacing helpers of the past are not needed at this time.  Divider, Image, Card were using it.  We should revisit spacing as part of a cross-cutting feature probably and figure out how to build that into the design system if its needed.  Chris may have some additional thoughts here as well.

Reply:
@awentzel I think we'll want a default here for vertical spacing and we may want to include any of the styles pertaining to a css reset or at least ensure that everything works across all browsers:

`
box-sizing: content-box;
height: 0;
margin-top: 12px;
margin-bottom: 12px;
border-top: 1px solid $color-foreground-20;
`

We can come back in and setup that spacing to be dynamic or based on a consistent utility